--- a/quantum/__init__.py
+++ b/quantum/__init__.py
@@ -15,5 +15,9 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-from pkgutil import extend_path
-__path__ = extend_path(__path__, __name__)
+try:
+        import pkg_resources
+        pkg_resources.declare_namespace(__name__)
+except ImportError:
+    import pkgutil
+    __path__ = pkgutil.extend_path(__path__, __name__)
--- /dev/null
+++ b/quantum/plugins/__init__.py
@@ -0,0 +1,6 @@
+try:
+    import pkg_resources
+    pkg_resources.declare_namespace(__name__)
+except ImportError:
+    import pkgutil
+    __path__ = pkgutil.extend_path(__path__, __name__)
--- a/setup.py
+++ b/setup.py
@@ -120,4 +120,5 @@
             'quantum-server = quantum.server:main',
         ]
     },
+    namespace_packages=['quantum',],
 )
